internal/neofs/neofs.go

```diff
@@ -102,12 +102,14 @@ func (x *NeoFS) Container(ctx context.Context, idCnr cid.ID) (*container.Contain
 	return res, nil
 }
 
+var basicACLZero acl.Basic
+
 // CreateContainer implements neofs.NeoFS interface method.
 //
 // If prm.BasicACL is zero, 'eacl-public-read-write' is used.
 func (x *NeoFS) CreateContainer(ctx context.Context, prm layer.PrmContainerCreate) (*cid.ID, error) {
-	if prm.BasicACL == 0 {
-		prm.BasicACL = acl.EACLPublicBasicRule
+	if prm.BasicACL == basicACLZero {
+		prm.BasicACL = acl.PublicRW
```

Comment:
I guess `PublicRW` is a constant for eACL enabled public container. Just wonder, what is the constant for eACL disabled public container?

Reply:
That's my mistake, fixed to `PublicRWExtended`.